tests/test_contact_specific_functions.py

```diff
@@ -233,117 +212,49 @@ def test_calculate_contact_forces_rod_rod():
 
     "Testing function with analytically verified values"
 
-    tol = 1e-5
-
-    "initializing rod 1 parameters"
-    rod_one_position_collection = np.array([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
-    rod_one_radius_collection = np.array([1.3, 0.5])
-    rod_one_length_collection = np.array([2.1, 0.9])
-    rod_one_tangent_collection = np.array([[0, -0.29], [-0.029, 1.12], [-1.04, -1.165]])
-
-    rod_one_internal_forces = np.array(
-        [
-            [0.59155762, -0.0059393, 0.39725841],
-            [-2.45916096, 0.81593963, 1.19209269],
-            [1.46996099, -0.52774132, -0.08287433],
-        ]
-    )
-    rod_one_external_forces = np.array(
-        [
-            [-1.55308218, 1.00714652, 2.04033497],
-            [-0.35890683, -0.22667242, -0.31385005],
-            [-0.66969949, -0.67150666, 0.17822428],
-        ]
-    )
-
-    rod_one_velocity_collection = np.array(
-        [
-            [-0.91917535, -1.04929632, -0.32208191],
-            [0.13459384, -1.77261144, -0.28656239],
-            [1.63574433, 0.31741648, 1.43917516],
-        ]
-    )
-
-    "initializing rod 2 parameters"
-    rod_two_position_collection = np.array(
-        [[1.0000001, 2.0000034, 3], [4, 5, 6], [7, 8, 9]]
-    )
-    rod_two_radius_collection = np.array([2.3, 0.4])
-    rod_two_length_collection = np.array([1.5, 1.0])
-    rod_two_tangent_collection = np.array([[0, -0.39], [-0.29, 1.00], [-1.0, -0.165]])
-
-    rod_two_internal_forces = np.array(
-        [
-            [-0.46552762, 1.16896583, 1.06695832],
-            [0.35104219, 0.5868144, -0.79542854],
-            [1.90896989, -1.7709093, 1.21209849],
-        ]
-    )
-    rod_two_external_forces = np.array(
-        [
-            [0.12615472, -0.12539237, -1.01333332],
-            [0.7193244, 1.12085914, 1.57535336],
-            [1.79006353, 0.20498294, 1.11384582],
-        ]
-    )
+    rod_one = MockRod()
 
-    rod_two_velocity_collection = np.array(
-        [
-            [-1.09200111, -0.87381467, 0.44073417],
-            [0.93961598, 1.25513012, 0.09103194],
-            [1.02214026, -0.78790631, -0.74019659],
-        ]
-    )
+    rod_two = MockRod()
+    """Placing rod two such that its first element pentrates the second element of rod one
+    by 1.0 unit in the x-direction."""
+    rod_two.position_collection = np.array([[2.5, 3.5], [0, 0], [0, 0]])
```

Comment:
update as @armantekinalp said here too, make sure the element lengths are correct